frontend/ui/widget/textviewer.lua

```diff
@@ -161,7 +133,7 @@ function TextViewer:init()
         show_parent = self,
     }
 
-    local textw_height = self.height - titlew:getSize().h - separator:getSize().h - button_table:getSize().h
+    local textw_height = self.height - titlebar:getSize().h - button_table:getSize().h
```

Comment:
I added a `titlebar:getHeight()` that returns the same value than `titlebar:getSize().h` , but makes it slightly more readable.